src/components/UserAccess/UserAccessForm/UsernameSection.tsx

```diff
@@ -111,9 +107,14 @@ export const UsernameSection: React.FC<React.PropsWithChildren<Props>> = ({ disa
                 {error}
               </HelperTextItem>
             ) : validHelpText ? (
-              <HelperTextItem variant="success" hasIcon>
-                {validHelpText}
-              </HelperTextItem>
+              <>
+                <HelperTextItem variant="success" hasIcon>
+                  {validHelpText}
+                </HelperTextItem>
+                <HelperTextItem variant="warning" hasIcon>
+                  We do not validate whether the username is a valid Konflux user.
+                </HelperTextItem>
```

Comment:
We should show an inline alert at the top of page. Something like this https://v5-archive.patternfly.org/components/alert#inline-alerts-variants, Get a UX review on this and text for it. The alert should be dismissible, and once dismissed, it should not reappear. Store its state in local storage.